sources/packages/web/src/components/common/OfferingSummary.vue

```diff
@@ -47,8 +46,8 @@
     ></Column>
     <Column :field="OfferingSummaryFields.StudyDates" header="Study Dates">
       <template #body="slotProps">
-        {{ dateOnlyLongString(slotProps.data.studyStartDate) }} -
-        {{ dateOnlyLongString(slotProps.data.studyEndDate) }}
+        {{ originalDateOnlyLongString(slotProps.data.studyStartDate) }} -
```

Comment:
Do we really need this formatter? I believe that the problem is that we are sending the "date only" alongside with the time and this time is also defined in UTC.
For designation agreement we faced the same issue. If the API returns only the date part there will be no issue in the the client application.

Reply:
@ann-aot, please take a look in the code below. We should not send TIME with a filed that in ONLY DATE. If you follow the below code there will be no issues with date.

![image](https://user-images.githubusercontent.com/61259237/153688502-7bec813a-f896-49e6-ac94-67df7e35ac76.png)